src/tests/suite/daos_nvme_recovery.c

```diff
@@ -201,22 +192,6 @@ nvme_test_verify_device_stats(void **state)
 
 	print_message("LOG FILE = %s\n", log_file);
 
-	/** Prepare records **/
-	if (arg->pool.pool_info.pi_nnodes < 2)
-		obj_class = DAOS_OC_R1S_SPEC_RANK;
-	else
-		obj_class = DAOS_OC_R2S_SPEC_RANK;
-
-	oid = dts_oid_gen(obj_class, 0, arg->myrank);
-	oid = dts_oid_set_rank(oid, 0);
-	ioreq_init(&req, arg->coh, oid, DAOS_IOD_ARRAY, arg);
-	memset(data_buf, 'a', 100);
-
-	/** Insert record **/
-	print_message("Insert single record with 100 extents\n");
-	insert_single_with_rxnr("dkey", "akey", idx, data_buf,
-				1, 100, DAOS_TX_NONE, &req);
-
```

Comment:
This can be left in the test, so faulty reaction will have something to rebuild when the device is marked as faulty. I was just suggesting to remove the additional insert fresh record section when rank < 1, and also the record verification at the end

Reply:
ok I will update the code in next PR.